### PR TITLE
Followup edits to PR 4674

### PR DESCRIPTION
**Pull request by bfallonf**
For: #4674 

cc: @pecameron 

Probably ok for peer review, @openshift/team-documentation thanks.

### CODE REVIEW

File: admin_guide/high_availability.adoc [hunk 1]
```diff
@@ -33,9 +33,9 @@ unavailable or if the service that *Keepalived* is watching does not respond,
 the VIP is switched to another host from the set. Thus, a VIP is always serviced
 as long as a host is available.
 
-When a host running *Keepalived* passes the *check* script it becomes available to
-become the *MASTER* based on its priority and the priority of the current *MASTER*.
-The configurable xref:vrrp-preemption[preemption strategy] determines what to do.
+When a host running *Keepalived* passes the *check* script, the host can become
+the *MASTER* based on its priority and the priority of the current *MASTER*, as
+determined by the xref:vrrp-preemption[preemption strategy].
```
Comment (mburke5678) [inline admin_guide/high_availability.adoc]: Why *MASTER*? This formatting is not used further down in a similar context:
"This enables any of the ipfailover nodes to become the master..."

File: admin_guide/high_availability.adoc [hunk 2]
```diff
@@ -218,7 +218,16 @@ link:http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-g
 the AWS console].
 ====
 
-As an administrator, you can configure ipfailover on an entire cluster, or on a subset of nodes, as defined by the label selector. You can also configure multiple IP failover deployment configurations in your cluster,  where each one is independent of the others. The `oc adm ipfailover` command creates a ipfailover deployment confiugration which ensures that a failover pod runs on each of the nodes matching the constraints or the label used. This pod runs link:http://www.keepalived.org/[*Keepalived*] which uses VRRP (Virtual Router Redundancy Protocol) among all the *Keepalived* daemons to ensure that the service on the watched port is available, and if it is not, *Keepalived* will automatically float the VIPs.
+As an administrator, you can configure ipfailover on an entire cluster, or on a
+subset of nodes, as defined by the label selector. You can also configure
+multiple IP failover deployment configurations in your cluster, where each one
+is independent of the others. The `oc adm ipfailover` command creates a
+ipfailover deployment confiugration which ensures that a failover pod runs on
```
Comment (mburke5678) [inline admin_guide/high_availability.adoc]: s/a ipfailover/an ipfailover

File: admin_guide/high_availability.adoc [hunk 3]
```diff
@@ -404,38 +413,36 @@ displayed in decimal (`493`).
...
-When a host leaves the *FAULT* state by passing the check script it can become *MASTER*.
-If the new host has lower priority than the current *MASTER* it becomes a *BACKUP*. If
-it has a higher priority the preemption strategy determines what to do. The *nopreempt*
-strategy does not move *MASTER* from the lower priority host to the higher priority host.
-With *preempt 300*, the default, *keepalived* waits the specified 300 seconds and moves
-*MASTER* to the higher priority host.
+When a host leaves the *FAULT* state by passing the check script, the host
+becomes a *BACKUP* if the new host has lower priority than the current *MASTER*.
+However, if it has a higher priority, the preemption strategy determines it's
+role in the cluster.
```
Comment (mburke5678) [inline admin_guide/high_availability.adoc]: Same here with BACKUP and MASTER. I see where there are apparently MASTER and BACKUP states; but here, MASTER is not referring to the state, but the "type" of system, right?

File: admin_guide/high_availability.adoc [hunk 4]
```diff
@@ -404,38 +413,36 @@ displayed in decimal (`493`).
...
+When a host leaves the *FAULT* state by passing the check script, the host
+becomes a *BACKUP* if the new host has lower priority than the current *MASTER*.
+However, if it has a higher priority, the preemption strategy determines it's
+role in the cluster. 
 
-Preemption can be specified when ipfailover is created through the --preemption-strategy
-option or by editing the *OPENSHIFT_HA_PREEMPTION* environment variable.
+The *nopreempt* strategy does not move *MASTER* from the lower priority host to
+the higher priority host. With *preempt 300*, the default, *keepalived* waits
+the specified 300 seconds and moves *MASTER* to the higher priority host.
```
Comment (mburke5678) [inline admin_guide/high_availability.adoc]: MASTER! MASTER!

**Comment by mburke5678** [comment]
@bfallonf Couple of nitty things. LGTM

**Comment by pecameron** [comment]
MASTER, BACKUP, and FAULT are states that a host can enter as the result 
of vrrp negotiation.

**Comment by bfallonf** [comment]
Thanks @mburke5678 . As per @pecameron 's comment, I made it more "in the `*MASTER*` state, etc.

@openshift/team-documentation anything else for this PR?

**Comment by bfallonf** [comment]
Similar markup [here](https://docs.openshift.com/container-platform/3.5/admin_guide/high_availability.html#check-notify).

**Comment by ahardin-rh** [comment]
@bfallonf Looks like this needs a rebase to merge it and unblock the original PR from getting into the OCP 3.6 release. 

cc @vikram-redhat

**Comment by bfallonf** [comment]
Thanks @ahardin-rh . Done and merging...

**Comment by ahardin-rh** [comment]
@bfallonf Thanks!

**Pull request merged**